unit_test/entities/test_esdl_object.py

```diff
@@ -173,6 +173,50 @@ def test_get_return_temperature_out(self):
         # Assert
         self.assertEqual(return_temperature, 313.15)
 
+    def test_get_supply_temperature_consumer(self):
```

Comment:
I don't know whether this test will cover what is required. You changed the input/output defintion of the Consumer/Producer asset but the tests did not fail; does this test now ensure that when you change that defintion this test will fail?

Reply:
Basically as I checked we do not have proper test for the controller to entity method. These should be added I created new issue for that. #191